.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-    "restructuredtext.confPath": "${workspaceFolder}/source",
+    "restructuredtext.confPath": "${workspaceFolder}\\source",
```

Comment:
I don't think this should be necessary. `/` is a valid path separator on all three platforms, but `\\` is only valid on Windows.